.github/workflows/copilot-setup-steps.yml

```diff
@@ -0,0 +1,33 @@
+name: Copilot Setup Steps
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
```

Comment:
Will remove the PR trigger. Needed to add this, just so I can test the workflow once before merging.